Dvorak-Right-Handed/KeyboardView.swift

```diff
@@ -21,7 +21,7 @@ class KeyboardView: UIView {
     var deleteTimer: Timer?
     var regularKeys: [UIButton] = []
     let myFont = UIFont.systemFont(ofSize: 24.0)
-    
+        
```

Comment:
Consider eliminating these type of changes in future pull requests as they tend to clutter the review process.